site/content/docs/main/release-instructions.md

```diff
@@ -1,16 +1,103 @@
-
+## General notes
+- Please read the documented variables in each scrip to understand what they are for and how to properly format their values.
+- You will need to have a remote named `origin` pointing to the Velero repository so the scripts can work properly.
+- GA release: major and minor releases only. Example: 1.0 (major), 1.5 (minor).
+- Pre-releases: Any release leading up to a GA. Example: 1.4.0-beta.1, 1.5.0-rc.1
+- RC releases: Release Candidate, contains everything that is supposed to ship with the GA release. This is still a pre-release.
+
+## Preparing
+
+### Create release blog post (GA only)
+For each major or minor release, create and publish a blog post to let folks know what's new. Please follow these [instructions](Instructions-to-write-and-release-a-blog-post).
+
+### Changelog and Docs PR
+#### Troubleshooting
+- `You don't have enough free space in /var/cache/apt/archives/` when running `make serve-docs`: run `docker system prune`.
+
+#### Steps
+1.  If it doesn't already exist: in a branch, create the file `changelogs/CHANGELOG-<major>.<minor>.md` by copying the most recent one.
+2.  Update the file `changelogs/CHANGELOG-<major>.<minor>.md`
+	- Run `make changelog` to generate a list of all unreleased changes.
+    - Copy/paste the output into `CHANGELOG-<major>.<minor>.md`, under the "All Changes" section for the release. 
+	- You *may* choose to tweak formatting on the list of changes by adding code blocks, etc.
+	- 	Update links at the top of the file to point to the new release version
+3.  Update the main `CHANGELOG.md` file to properly reference the release-specific changelog file
+	- Under "Current release": 
+	    - Should contain only the current GA release.
+    - Under "Development release": 
+	    - Should contain only the latest pre-release
+	    - Move any prior pre-release into "Older releases"
+4. GA Only: Remove all changelog files from `changelogs/unreleased`.
+5. Generate new docs
+	- Run `make gen-docs`, passing the appropriate variables. Examples:
+		a) `VELERO_VERSION=v1.5.0-rc.1 NEW_DOCS_VERSION=v1.5.0-rc.1 make gen-docs`.
+		b) `VELERO_VERSION=v1.5.0 NEW_DOCS_VERSION=v1.5 make gen-docs`).
+	- Note: `PREVIOUS_DOCS_VERSION=<doc-version-to-copy-from>` is optional; when not set, it will default to the latest doc version.
+6. Clean up when there is an existing set of pre-release versioned docs for the version you are releasing
+	- Example: `site/content/docs/v1.5.0-beta.1` exists, and you're releasing `v1.5.0-rc.1` or `v1.5`
+    - Remove the directory containing the pre-release docs, i.e. `site/content/docs/<pre-release-version>`.
+    - Delete the pre-release docs table of contents file, i.e. `site/data/<pre-release-version>-toc.yml`.
+    - Remove the pre-release docs table of contents mapping entry from `site/data/toc-mapping.yml`.
+    - Remove all references to the pre-release docs from `site/config.yml`.
+7. Create/update the "Upgrade to $major.minor" page.
```

Comment:
Could we include an example path to this file, e.g. `site/content/docs/<version>/upgrade-to-<major>.<minor>.md`? Does it need to be updated in both the versioned *and* main docs folders?

Reply:
Good question. I added "This needs to be done in both the versioned and the `main` folders.".